InternetHospitalWebUI/src/app/Components/Authorization/sign-up/sign-up.component.ts

```diff
@@ -7,9 +10,21 @@ import { Component, OnInit } from '@angular/core';
 })
 export class SignUpComponent implements OnInit {
 
-  constructor() { }
+  constructor(private service: AuthorizationService) { }
 
   ngOnInit() {
   }
 
+  onClear() {
+    this.service.form.reset();
+    this.service.initializeFormGroup();
+  }
```

Comment:
What the difference between `form.reset()` and `service.initializeFormGroup()` method where you simply assign empty values for form's inputs? Does not `form.reset()` make the same action?